glfw/src/Window.zig

```diff
@@ -180,8 +183,10 @@ pub const Hint = enum(c_int) {
     /// Context flush-on-release hint.
     context_release_behavior = c.GLFW_CONTEXT_RELEASE_BEHAVIOR,
 
-    /// Context error suppression hint.
-    context_no_error = c.GLFW_CONTEXT_NO_ERROR,
+    // TODO: The docs don't specify GLFW_CONTEXT_REVISION as a hint, only as an attribute; is there a reason this is here? Commented to make current
+    // draft work, which is mostly copying from the docs
```

Comment:
This one is a hint, and is actually [listed in the docs](https://www.glfw.org/docs/latest/window_guide.html#GLFW_CONTEXT_NO_ERROR_hint)

Reply:
@silversquirl Right, I didn't see that. I was looking [at this list](https://www.glfw.org/docs/latest/window_guide.html#window_hints_values), which lists hints, their default values and possible values. It's not in that list, and I can't see anywhere else that would specify a default value for it; maybe it could default to `!std.debug.runtime_safety`? (set to true in release modes).

Reply:
Since we handle errors through Zig error unions, I'd say definitely default it to false. Possibly even remove it from the struct altogether, since it's quite an unsafe option to use - at the very least put a doc comment informing about the dangers.

Reply:
@silversquirl Alright, I suppose that makes sense. I'll amend it now and leave this as resolved.